RptToXml/Program.cs

```diff
@@ -21,10 +21,14 @@ static void Main(string[] args)
 			string rptPathArg = args[0];
 			var rptPaths = new List<string>();
 			int ignoreErrFlag = 0;
+			int stdOut = 0;
 
 			if (args.Contains("--ignore-errors"))
 				ignoreErrFlag = 1;
 
+			if (args.Contains("--stdout"))
+				stdOut = 1;
+
 			if ("-r".Equals(rptPathArg, StringComparison.InvariantCultureIgnoreCase))
 			{
 				if (args.Length > 1 + ignoreErrFlag)

```

Comment:
I know this PR isn't about ignore err, but shouldn't error ignore only apply to the try/catch around the actual report writing? Users still want to know about errors with how they are passing args I would think.